conformance_tests/sysman/test_sysman_device/src/test_sysman_device.cpp

```diff
@@ -466,6 +465,14 @@ void compute_workload(workload_thread_parameters *t_params) {
   lzt::destroy_module(module);
 }
 
+void set_reset_properties(zes_reset_properties_t &properties, ze_bool_t force,
```

Comment:
This can be wrapped as well in sysman_device_reset_ext(), @joshuaranjan what do you think. we can only pass reset type, force option along with device.

Reply:
Yes.